Harold/main.py

```diff
@@ -48,6 +48,10 @@ def main():
                         help="FIFO to communicate to mplayer with")
     parser.add_argument("--nobeep", "-n", action="store_true",
                         help="Disable beep")
+    parser.add_argument("--overrides", "-o", default="",
+                        help="allow users to be override others")
+    parser.add_argument("--blacklist", "-b", default="",
+                        help="blacklist certain usernames")
```

Comment:
I'd suggest you use `nargs="*"` instead of `default=""`. That way it returns a list and you don't have to handle splitting it below.